### PR TITLE
MRG, ENH: Add infant template downloader

### PR DESCRIPTION
**Pull request by larsoner**
Adds infant templates from [NeuroImage](http://www.sciencedirect.com/science/article/pii/S1053811920311678)

cc @christian-oreilly

### CODE REVIEW

**Comment by christian-oreilly** [comment]
Thanks for this PR @larsoner. Checking out a copy of this PR and trying `mne.datasets.fetch_infant_template("2wk")` works well on my side! Next week, I need to show a student how to do source reconstruction on infants EEG so I will test this feature more thoroughly at this time. I will let you know if I observe any issues during this process.

**Comment by larsoner** [comment]
> I need to show a student how to do source reconstruction on infants EEG

@christian-oreilly feel free to comment on the new narrative doc with better wording, etc:

https://24496-1301584-gh.circle-artifacts.com/0/dev/auto_tutorials/source-modeling/plot_eeg_no_mri.html#infant-mri-surrogates

If you know of a publicly available infant EEG dataset that we could use, then it could probably be a nice standalone tutorial, too.

**Comment by larsoner** [comment]
@agramfort I think this one is good to go. I and a couple other folks at ILABS have used it thus far. It tracks version 0.1 of the dataset, and if there are further changes, we can update our definitions/lists.

File: tutorials/source-modeling/plot_eeg_no_mri.py [hunk 1]
```diff
@@ -79,3 +84,61 @@
...
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^
+# To use an infant head model for M/EEG data, you can use
+# :func:`mne.datasets.fetch_infant_template` to download an infant template:
+
+subject = mne.datasets.fetch_infant_template('6mo', subjects_dir, verbose=True)
+
+##############################################################################
+# It comes with several helpful built-in files, including a 10-20 montage
+# in the MRI coordinate frame, which can be used to compute the
+# MRI<->head transform ``trans```:
```
Comment (drammock) [inline tutorials/source-modeling/plot_eeg_no_mri.py:112]: extra backtick

```suggestion
# MRI<->head transform ``trans``:
```

**Review by agramfort — APPROVED** [review]
merge when happy

thx

**Pull request merged**